bin/bst-proxy.ts

```diff
@@ -69,6 +71,7 @@ program
     .option("--bstHost <bstHost>", "The host name of the BST server")
     .option("--bstPort <bstPort>", "The port of the BST server", parseInt)
     .option("--pithy", "Disables verbose diagnostics")
+    .option("--verbose", "Enable verbose diagnostics (activated by default)")
```

Comment:
There is no argument to disable an specific option from the help output, but we have two options apart from leaving this message as it is.
1. We can manually generate the help output, there's a commander option to output our own help instead of the default generated one with the options.
2. We can remove the --verbose option but add the allowUnknowOptions flag for the command, that one will do nothing for any unknown option passed to the command.